tests/conftest.py

```diff
@@ -90,6 +92,11 @@ def init_db(qapp, qtbot, tmpdir_factory):
     source_dir = SourceFileModel(dir='/tmp/another', repo=new_repo, dir_size=100, dir_files_count=18, path_isdir=True)
     source_dir.save()
 
+    signals.post_save.disconnect(receiver=qapp.main_window.miscTab.on_setting_update, sender=SettingsModel)
+    # reconnect autostart signal
```

Comment:
Can elaborate why this is needed (because of peewee bug) and link coleifer/peewee#2687?

Reply:
> Can elaborate why this is needed (because of peewee bug) and link [coleifer/peewee#2687](https://github.com/coleifer/peewee/issues/2687)?

Sorry! I should have added it before that is a good practice 😄 

Done! Also added the link to the issue 👍🏼